src/command_modules/azure-cli-monitor/azure/cli/command_modules/monitor/_help.py

```diff
@@ -207,3 +207,57 @@
     type: group
     short-summary: Manage activity logs.
 """
+
+helps['monitor action-group'] = """
+    type: group
+    short-summary: Manage action groups
+"""
+
+helps['monitor action-group list'] = """
+    type: command
+    short-summary: List action groups under a resource group or the current subscription
+    parameters:
+        - name: --resource-group -g
+          type: string
+          short-summary: Name of the resource ground under which the action groups are being listed. If it is omitted,
+                         all the action groups under the current subscription are listed.
+"""
+
+helps['monitor action-group show'] = """
+    type: command
+    short-summary: Show the details of an action group
+"""
+
+helps['monitor action-group create'] = """
+    type: command
+    short-summary: Create a new action group
+    parameters:
+        - name: --add-receiver -a
+          short-summary: Add receivers to the action group during the creation
+          long-summary: |
+            Usage:   --add-receiver TYPE NAME [ARG ...]
```

Comment:
Why not maintain consistency with the alert create command and have this simply be `--action`?  Receiver is a very bizarre term and as a user I would have no idea what that means (actually... as a developer I have no idea what that means).

Also, since you can ONLY add actions during create, I would recommend making this simply `--action` for create vs. `--add/remove-action` for update.

Reply:
ok